site/gatsby-site/src/templates/cite.js

```diff
@@ -125,7 +132,7 @@ function CitePage(props) {
       </AiidHelmet>
 
       <div className={'titleWrapper'}>
-        <StyledHeading>{metaDescription}</StyledHeading>
+        <StyledHeading>{locale == 'en' ? metaTitle : defaultIncidentTitle}</StyledHeading>
```

Comment:
I think Sean wanted it to have the incident ID *and* the title.

Reply:
Yes please!

Reply:
@smcgregor This way sounds good to you?
![image](https://user-images.githubusercontent.com/6564809/181837303-a367336e-9c94-4f03-8a4a-df8dbcb1e9d5.png)

Reply:
Looks good to me!